docs/stats/index.rst

```diff
@@ -13,9 +13,17 @@ in astronomy and astropy.
 Getting Started
 ===============
 
-The current tools are fairly self-contained, and include relevant examples in
+Most tools are fairly self-contained, and include relevant examples in
 their docstrings.
 
+Documentation
```

Comment:
In most of the other doc sections, the "more detail" section like this is titled "Using <packagename>".  See, e.g. [the coordinates index page](https://github.com/astropy/astropy/blob/master/docs/coordinates/index.rst). I think this should be changed to 
```
Using `astropy.stats` 
=====================
```